packages/create-pantheon-decoupled-kit/src/templates/gatsby-wp/package.json.hbs

```diff
@@ -1,13 +1,5 @@
 {
-	"name": "{{kebabCase appName}}",
-	"version": "0.1.0",
-	"private": true,
-	"license": "GPL-3.0-or-later",
-	"description": "",
-	"homepage": "",
-	"bugs": "",
-	"author": "",
-	"repository": "",
+	{{> sharedPkgJsonFields}}
```

Comment:
Only noticed one tiny thing when reviewing this. The introduction of the partial here is resulting in the scripts section not starting on a new line. Still valid JSON, but something we should clean up.

<img width="875" alt="Screenshot at Feb 13 10-32-20" src="https://user-images.githubusercontent.com/889478/218518664-1a0aa070-7b2c-443b-901b-b6eb9d9c79eb.png">

Reply:
nice catch! this should be fixed now.